ignite/handlers/param_scheduler.py

```diff
@@ -432,7 +470,8 @@ def print_lr():
     .. versionadded:: 0.4.5
     """
 
-    def get_param(self) -> float:
+    def _get_cycle_param(self) -> float:
```

Comment:
please rename `get_param -> _get_cycle_param` in `CosineAnnealingScheduler` class as well.